newgame/src/resources/types_template.txt

```diff
@@ -0,0 +1,28 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
+pub struct $CLASSNAME {
+    // Fill in the high-level fields of your game here
+}
+
+// Put the rest of the structs for your game here
+
+#[derive(Clone, Serialize, Deserialize, JsonSchema)]
+pub struct StateCore {
```

Comment:
This is a *huge* bikeshed, but definitely for new games, I think this will make more sense to new people as "FrameState" rather than the silly "StateCore" that's my fault. #12.

OK to punt on this since I have #12 open.

Reply:
This should go into another pull request.